build/templates/session.py.mako

```diff
@@ -234,13 +234,13 @@ if grpc_supported:
         {
             'default_value': None,
             'direction': 'in',
-            'documentation': { 'description': 'MeasurementLink gRPC channel' },
+            'documentation': { 'description': 'MeasurementLink gRPC options' },
             'enum': None,
             'is_repeated_capability': False,
             'is_session_handle': False,
-            'python_name': '_grpc_channel',
+            'python_name': '_grpc_options',
             'size': {'mechanism': 'fixed', 'value': 1},
-            'type_in_documentation': 'grpc.Channel',
+            'type_in_documentation': 'GrpcOptions',
```

Comment:
Since the user will have to construct this manually, we should probably be more clear where they need to get this from: `module_name + '.grpc_options.GrpcOptions'`